submissions/tobiToyo/PHP/weather.php

```diff
@@ -0,0 +1,55 @@
+<?php
+  
+  function weather ($temp){
+      if ($temp <= 20) {
+        echo "It is freezing today";
+      }
+      elseif ($temp > 20 && $temp < 30) {
+        echo "The weather is just perfect.";
+      }
+      elseif ($temp >= 30 && $temp < 40) {
+        echo "It's so hot today.";
+      }
+      else {
```

Comment:
Replace with this instead:
 } elseif ($temperature >= 40) {
    echo "Am I in the Sahara Desert?";
} else {
    echo "Please enter the day's temperature";
}